example_files/visualization/input_visualization_feature.html

```diff
@@ -467,13 +466,15 @@ <h2>Annual End Uses</h2>
             $scope.monthlyNetChartOptions.chart['yDomain'] = [_.round(monthlyNetYMin), _.round(monthlyNetYMax)];
           }
           
+          var total_value_feature = 0
           // annual end use
           if(feature['complete_simulation'] == true){
           $scope.annualEndUseChartData[feature.name] = [];
           // Find all applicable end uses
           _.forEach(applicableEndUseKeys, function (endUseKey){
               var endUseValue = feature.annual_values[endUseKey]
-
+              
+              total_value_feature += endUseValue
```

Comment:
@kflemin I have added the `total_value_feature` as the local average for each feature